plugins/Status.cpp

```diff
@@ -87,7 +87,7 @@ bool BedrockPlugin_Status::peekCommand(BedrockNode* node, SQLite& db, BedrockNod
         content["state"] = SQLCStateNames[node->getState()];
         content["hash"] = node->getHash();
         content["commitCount"] = SToStr(node->getCommitCount());
-        content["version"] = SVERSION;
+        content["version"] = _args ? (*_args)["version"] : "";
```

Comment:
Yeah, when I chose this, I was thinking of our own version strings, which are sha1 hashes, and unlikely to conflict, but there's no requirement for plugin authors to use those versions. I'll update this.